### PR TITLE
More uses of kzd_app

### PR DESCRIPTION
**Pull request by fenollp**
also formatting, some refactor into LCs & less funs.

### CODE REVIEW

File: applications/crossbar/src/cb_apps_util.erl [hunk 1]
```diff
@@ -298,33 +281,20 @@ get_plan_apps(ServicePlan) ->
...
-            lager:debug("excluding app ~s(~s)", [AppName, AppId]),
-            Acc;
-        'undefined' ->
-            lager:debug("excluding app ~s(~s)", [AppName, AppId]),
-            Acc;
-        AppJObj ->
-            lager:debug("including app ~s(~s) from plan", [AppName, AppId]),
-            [kz_json:delete_key(<<"published">>, AppJObj)|Acc]
-    end.
+    [kz_json:delete_key(<<"published">>, MaybeAppJObj)
```
Comment (k-anderson) [inline applications/crossbar/src/cb_apps_util.erl]: @fenollp Howdy, this PR looks good except for this... I would not describe this as more readable.  Why the change to list comprehensions?

Comment (fenollp) [inline applications/crossbar/src/cb_apps_util.erl]: Ah yes I agree I have made a bad choice here. Will revert

**Comment by fenollp** [comment]
@k-anderson done

File: core/kazoo_documents/src/kzd_app.erl [hunk 2]
```diff
@@ -225,18 +225,26 @@ account_id(JObj) ->
 %% @doc
 %% @end
 %%--------------------------------------------------------------------
--spec users(kz_json:object()) -> api_binaries().
+-spec users(kz_json:object()) -> api_objects().
```
Comment (lazedo) [inline core/kazoo_documents/src/kzd_app.erl]: @fenollp isn't users a list of Ids ? `is_authorized` in cb_apps_util does a `lists:member(UserId, UserIds);`
